kubernetes/assets/monitors/monitor_pod_imagepullbackoff.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "[kubernetes] Pod {{pod_name.name}} is ImagePullBackOff on namespace {{kube_namespace.name}}",
+	"type": "query alert",
+	"query": "max(last_10m):max:kubernetes_state.container.waiting{reason:imagepullbackoff} by {kube_namespace,pod_name} >= 1",
+	"message": "pod {{pod_name.name}} is ImagePullBackOff on {{kube_namespace.name}}",
```

Comment:
same thing. could we recommend some next steps?

Reply:
This could happen for several reasons, could be a bad image path/tag or the creds for pulling images are not configured properly. There is no one definitive action to take here. Sometimes we have seen this happen with bad networking on the node as well